docs/getting-started/release-notes/v2_9_0.md

```diff
@@ -30,10 +50,45 @@ Zowe Version 2.9.0 contains the bug fixes that are described in the following to
+
+- Fixed a logic error in the `config list` command that caused unwanted behavior when a positional argument and the `--locations` option were both passed in the command.
+
+- Reduced file loading time by searching for command definitions with the `fast-glob` NPM module instead of the `glob` module.
+
+- Removed validation of the deprecated `pluginHealthCheck` property because it was an unused feature. ([#980](https://github.com/zowe/imperative/issues/980))
+
 ### Zowe CLI
 
+- Enabled ANSI in Windows-based terminals to better display progress bars when using daemon mode. ([#1701](https://github.com/zowe/zowe-cli/issues/1701))
+
+- Changed daemon to spawn as its own process to avoid unintentional termination of the daemon mode. ([#1241](https://github.com/zowe/zowe-cli/issues/1241), [#1277](https://github.com/zowe/zowe-cli/issues/1277), [#1309](https://github.com/zowe/zowe-cli/issues/1309))
+
+- Fixed `--secondary` data set allocation option being specified as `1` unit on `BLANK` type data sets with the `zowe files create data-set` command. ([#1595](https://github.com/zowe/zowe-cli/issues/1595))
+
+- Fixed `--range` option so it is not ignored on the `zowe files view uss-file` command.
+
+- Fixed `--binary` option ignored by commands that upload and download USS directories when the `.zosattributes` file is used.
+
+- Fixed `--include-hidden` option ignored by the `zowe files upload dir-to-uss` command when it was used without the `--recursive` option.
+
+- Implemented several updates for technical currency. Updated Imperative to allow for special handling of chalk and coloring in daemon client. Updated `imperative` to fix undesired behavior in the `zowe config list` command in certain situations. Updated `tar` dependency.
+
+### IBM Db2 Database Plug-in for Zowe CLI
+
+- Updated `ibm_db` dependency for better support with Node.js 18.
+
 ### Zowe Explorer
 
+- Fixed issue where the **Disable Validation for Profile** context menu option did not update to **Enable Validation for Profile** after use. ([#1897](https://github.com/zowe/vscode-extension-for-zowe/issues/1897))
+- Removed "/" characters in `path.join()` calls. ([#2172](https://github.com/zowe/vscode-extension-for-zowe/issues/2172))
+- Fixed issue where user was not able to view job spool file with the same DD name in different steps because of duplicated local file name. ([#2279](https://github.com/zowe/vscode-extension-for-zowe/issues/2279))
+- Fixed issue where user was not able to view job spool file from jobs with duplicated step names because of duplicated local file name. ([#2315](https://github.com/zowe/vscode-extension-for-zowe/issues/2315))
+- Fixed issue with Windows path when uploading file to data set. ([#2323](https://github.com/zowe/vscode-extension-for-zowe/issues/2323))
+- Fixed an issue with mismatch etag error returned not triggering the diff editor and possible loss of data due to the issue. ([#2277](https://github.com/zowe/vscode-extension-for-zowe/issues/2277))
+
```

Comment:
I think we will also need to add:

Bug:
- Fixed issue where refreshing views collapsed the respective trees. [#2215](https://github.com/zowe/vscode-extension-for-zowe/issues/2215)

New feature:
- Added support for custom credential manager extensions in Zowe Explorer [#2212](https://github.com/zowe/vscode-extension-for-zowe/issues/2212)

I think these will need to be added to the changelog on the 2.9.0 release in the Zowe Explorer repo. Not sure I understand how the 2.9.0 PR is supposed to work in terms of the changelog.